onnxruntime/contrib_ops/cuda/bert/group_query_attention_helper.h

```diff
@@ -29,13 +29,13 @@ Status CheckInputs(const Tensor* query,
   //     query            (Q)       : (B, S, D)
   //     key              (K)       : (B, S+, D_kv)
   //     value            (V)       : (B, S+, D_kv)
+  ORT_UNUSED_PARAMETER(value);
```

Comment:
why not check the shape of value?

Reply:
Yufeng said I don't need to? https://github.com/microsoft/onnxruntime/pull/17920#discussion_r1367557306

Reply:
you can try test some inputs with invalid shape.

Reply:
If there is shape information in onnx graph, it will check. What if there is no shape information?

Reply:
Good point, I will add back in the checks